eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/equiz.compileDirective.js

```diff
@@ -0,0 +1,19 @@
+﻿(function (angular) {

```

Comment:
Rename the file according to https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md#naming